validator/src/main.rs

```diff
@@ -53,6 +57,12 @@ async fn main() {
 
     tokio::task::spawn(api_main());
 
+    let updater = Updater::new(
```

Comment:
This needs to be used on miners as well.

Reply:
line 308-312 in miner/src/main.rs, its already there